gym_solo/core/test_termination_conditions.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from gym_solo.core import termination
+
+class TestTimeBasedTermination(unittest.TestCase):
+    def test_attributes(self):
+      max_step_delta = 3
+      term = termination.TimeBasedTermination(max_step_delta)
+      self.assertEqual(max_step_delta, term.max_step_delta)
+      self.assertEqual(0, term.step_delta)
+
+    def test_reset(self):
+      max_step_delta = 3
+      term = termination.TimeBasedTermination(max_step_delta)
+
+      for i in range(max_step_delta):
+        term.is_terminated()
```

Comment:
maybe make this a `self.assertFalse`? Might as well ensure that there isn't some funny business happening in this loop.